model/components/prometheus.yml

```diff
@@ -0,0 +1,105 @@
+---
+components:
+
+  # prometheus
+  - id: prometheus
+    name: Prometheus
+    description: Prometheus is an open-source systems monitoring and alerting toolkit.
+    link: https://prometheus.io/
+    logo: prometheus
+    categories:
+      - agent
+    capabilities:
+      - tech:
+        - java
+        - go
+        - scala
+        - python
+        - ruby
+        - bash
+        - c++
+        - common-lisp
+        - elixir
+        - erlang
+        - haskell
+        - lua
+        - .net
+        - c#
+        - nodejs
+        - php
+        - rust
+      - aspects:
+        - metrics-app
+        - metrics-process
```

Comment:
we are missing `metrics-host` or `metrics-infrastructure` as node exporter does export host metrics. @AlexanderWert what name do you prefer?

Reply:
let's take the shorter one :-) metrics-host